### PR TITLE
Fix top banner hiding text

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
I don't know what happened with the CSS formatting so I fixed it... But the important commit is the second one: it prevents text to go under the top banner. For example: https://docs.rs/halt/0.3.0/halt/struct.Invalid.html#implementations, the title won't "disappear" under the banner.

cc @pietroalbini

### CODE REVIEW

**Comment by jyn514** [comment]
I didn't even realize this was a problem with docs.rs, I thought firefox was being wacky. Thanks for fixing this!

File: css/style.css [hunk 1]
```diff
@@ -13,12 +13,13 @@ html, button, input, select, textarea,
 
 div.rustdoc {
   font-family: "Source Serif Pro", Georgia, Times, "Times New Roman", serif;
-  padding: 10px 15px 20px 15px;
```
Comment (jyn514) [inline css/style.css]: It looks like taking out this 'padding' style makes the search bar overlap with the style button, at least in my browser.

image

Comment (GuillaumeGomez) [reply css/style.css]: Oh funny. Can you open an issue about this (and maybe even a PR?). I don't want to mix more things in this PR.

Comment (jyn514) [reply css/style.css]: I'm not sure I understand, do you mean in the main rust repo? If you just add the padding back in (like it was before) it works fine.

Comment (GuillaumeGomez) [reply css/style.css]: Oh crap! Completely missed the fact that I removed it. Good catch! Sorry for not understanding at first.

**Comment by GuillaumeGomez** [comment]
@jyn514 I have put back what I mistakenly removed.

**Comment by jyn514** [comment]
Great, thanks so much!

**Pull request merged**